### ISSUE TITLE
(Proposal) Not nearly enough details in your plan

### ISSUE THREAD

**Issue by aimirza** [UBC-MDS/DSCI_522_jchen9314_apchiodo#19]
> We will firstly get familiar with data set by performing some exploratory data analysis.

Like what? 
Misspelled firstly.

> Then, we will split data set into training and test sets. 

Split into what proportions? 

> Also, we will separate features of interest from target. 

How will you select "interesting" features?

> We will then apply decision tree classification to the training set using scikit-learn

What is _scikit-learn_? Seems like a package name, but what programming language. I know its python but please state it here :)

> and play with some hyperparameters.

Please be more concrete in your plan. What does "play with some parameters" mean? Please state how you plan to tune parameters and what parameters.

> Finally, we will evaluate how our model performed on the test set

What and how will you evaluate?

>  and tease out the most important physiochemical characteristics for classifying a good wine.

How? Using what method? 

> Summarize the data
> The output of the machine learning excercise will give us feature importance in determining wine quality, which we can show in tabular form. To visualize, we can create an ordered bar chart of each feature.

Sounds like a plan. 
Misspelled exercise.

**Comment by apchiodo** [comment]
I will address your comments in the order as they appear:

1.  By creating grouped summaries and visualizations (such as violin plots), we were able to explore the data in order to evaluate any discrepancies, limitations, and most importantly to help guide the project moving forward.  For example, we initially thought that with a ranking system between 0 and 10 (as a measure of wine quality and the classification that we would ultimately be trying to predict), we would have 11 categories to deal with.  In fact, we only had 6.  Furthermore, the data was not distributed evenly across the categories.

2. We addressed this in final report.

3. I think "features of interest" just meant features.  We will watch our word choices moving forward.

4.  We addressed this in final report.

5.  We evaluated the model performance using the model score, or accuracy, as determined by comparing how well the model worked on the test set (i.e. predicted correct labels).

6. As per https://scikit-learn.org/stable/modules/generated/sklearn.tree.DecisionTreeClassifier.html
> "The importance of a feature is computed as the (normalized) total reduction of the criterion brought by that feature. It is also known as the Gini importance."

7.  Addressed.
